vendors/marvell/boards/mw300_rd/CMakeLists.txt

```diff
@@ -179,8 +179,9 @@ set(
     "${mw320_dir}/sdk/src/boards"
     "${mw320_dir}/sdk/src/incl/platform/net/freertos-plus-tcp"
     "${AFR_KERNEL_DIR}/portable/GCC/ARM_CM4F"
-    "${AFR_ROOT_DIR}/libraries/3rdparty/mbedtls/include/"
-    "${AFR_ROOT_DIR}/libraries/3rdparty/mbedtls/include/mbedtls/"
+    "${AFR_ROOT_DIR}/libraries/3rdparty/pkcs11"
```

Comment:
My original intention was to not change the style of the file. Since the file had previously added those header files by adding the include path directly, I wanted to just update the include path instead of adding a new dependency. 

Although I agree adding a dependency is a cleaner and more maintainable approach so I have updated the CMake file. 

The change can be seen in [this commit](https://github.com/aws/amazon-freertos/pull/1881/commits/2969893cae38a923c6fdd749c0cf4eb659953a83).